__tests__/validators/or.test.js

```diff
@@ -0,0 +1,110 @@
+const Or = require('../../lib/validators/or')
+const Helper = require('../../__fixtures__/helper')
+
+test('should run subtasks', async () => {
+  const or = new Or()
+  const settings = {
+    do: 'or',
+    validate: [
+      {
+        do: 'milestone',
+        must_include: {
+          regex: 'Version 2'
+        }
+      }
+    ]
+  }
+  let validation = await or.validate(createMockContext({title: 'Version 1'}), settings)
+  expect(validation.status).toBe('fail')
+})
+
+test('should return output of second task if first fails', async () => {
+  const or = new Or()
+  const settings = {
+    do: 'or',
+    validate: [
+      {
+        do: 'milestone',
+        must_include: {
+          regex: 'Version 2'
+        }
+      },
+      {
+        do: 'milestone',
+        must_include: {
+          regex: 'Version 1'
+        }
+      }
+    ]
+  }
+  let validation = await or.validate(createMockContext({title: 'Version 1'}), settings)
+  expect(validation.status).toBe('pass')
+})
+
+test('should return output of first task to pass when multiple are given', async() => {
+  const or = new Or()
+  const settings = {
+    do: 'or',
```

Comment:
Is it possible to nest `do: 'or' ` ?
i.e 
``` 
do : or 
validate: 
  do : or
    validate: 
       do : A
       do : B
  do : C
```

Reply:
Should work, not sure why you’d do this though.

Reply:
You are right, the scenario i proposed doesn't make sense, sorry about that. My thought were to make sure if and when we have 'and' validator, we can do nested 'and' and 'or' to create more complex validations. I should've taken a deeper look at your code first before asking.